src/drive/web/modules/navigation/duck/reducer.js

```diff
@@ -35,11 +38,6 @@ import {
   OPEN_FOLDER_FROM_TRASH_FAILURE
 } from 'drive/web/modules/trash/actions'
 
-import {
-  OPEN_FOLDER_FROM_SHARINGS_FAILURE,
-  OPEN_FOLDER_FROM_SHARINGS,
-  OPEN_FOLDER_FROM_SHARINGS_SUCCESS
-} from 'drive/web/modules/drive/SharingsContainer'
```

Comment:
So we can remove them from `SharingsContainer` ?

Reply:
Yes, they are removed.